src/cdk/dialog/dialog.md

```diff
@@ -159,6 +159,14 @@ If you're using a `TemplateRef` for your dialog content, the data is available i
 </ng-template>
 ```
 
+If you're using a `DialogRef`, you can use:
```

Comment:
Should `DialogRef` here refer to a `TemplateRef`?

Reply:
Updated the wording to be more clear.